src/components/Header/header.jsx

```diff
@@ -189,6 +194,7 @@ const mapStateToProps = store => ({
   hook: store.hookState.currentHook,
   patientId: store.patientState.currentPatient.id,
   isCardDemoView: store.cardDemoState.isCardDemoView,
+  isSecuredSandbox: store.fhirServerState.accessToken,
```

Comment:
Does ```store.fhirServerState.accessToken``` store an access-token or a boolean?

Reply:
Its the access token itself, which should return either the object itself or null. I thought putting the value in `isSecuredSandbox` would make more sense in context in this component, rather than checking `if (hasAccessToken) { . . . }`